sdk/monitor/Azure.Monitor.Ingestion/README.md

```diff
@@ -147,6 +149,77 @@ Response response = await client.UploadAsync(
     RequestContent.Create(data)).ConfigureAwait(false);
 ```
 
+### Upload custom logs as IEnumerable
+
+You can upload logs using either the `LogsIngestionClient.Upload` or the `LogsIngestionClient.UploadAsync` method. In these two methods, logs are passed in a generic `IEnumerable` type. Additionally, there's an `UploadLogsOptions`-typed parameter in which a serializer, concurrency, and EventHandler can be set.
```

Comment:
[nit] > Additionally, there's an `UploadLogsOptions`-typed parameter in which a 

-typed parameter should be typed parameter?